pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -733,14 +697,14 @@ int socket__http2_filter(struct __sk_buff *skb) {
     // We have couple of interesting headers, launching tail calls to handle them.
     if (bpf_map_update_elem(&http2_iterations, &dispatcher_args_copy, iteration_value, BPF_NOEXIST) >= 0) {
         // We managed to cache the iteration_value in the http2_iterations map.
-        bpf_tail_call_compat(skb, &protocols_progs, PROG_HTTP2_FRAME_PARSER);
+        bpf_tail_call_compat(skb, &protocols_progs, PROG_HTTP2_HEADERS_PARSER);
     }
 
     return 0;
 }
 
-SEC("socket/http2_frames_parser")
-int socket__http2_frames_parser(struct __sk_buff *skb) {
+SEC("socket/http2_headers_parser")
```

Comment:
maybe it is a good time to document our current programs and the callers in order to make some order ?

Reply:
np

Reply:
I'll add documentation only for the new programs, and for the old ones I'll make that in a separate PR